src/commcare_cloud/environment/main.py

```diff
@@ -66,6 +66,10 @@ def check(self):
             self.proxy_config
         self.create_generated_yml()
 
+    @property
+    def is_dimagi_env(self):
+        return self.name in ["production", "staging", "india"]
```

Comment:
instead of doing this, what about having variable in the environment that determines whether to display the info? e.g. 'show_maintenance_info_on_deploy'?

Reply:
True, actually however it appears there is already a public var called `IS_DIMAGI_ENVIRONMENT` so I can use that. It is unlikely we need this on Dimagi envs since we do all the maintenance proactively, so an extra var is not super necessary.

Reply:
Or may be some it will be nice to have it enabled even if it is a Dimagi env such as PNA or SWISS. I will update